keys.go

```diff
@@ -548,6 +555,39 @@ func (c *Client) wrap(ctx context.Context, idOrAlias string, plainText []byte, a
 	return pt, ct, nil
 }
 
+// WrapWithKeyVersion function supports KeyVersion Details, PlainText and Cyphertext in response
+func (c *Client) WrapV2(ctx context.Context, idOrAlias string, plainText []byte, additionalAuthData *[]string) (*KeyActionResponse, error) {
+	keysActionReq := &KeysActionRequest{}
+	keyActionRes := &KeyActionResponse{}
+
+	if plainText != nil {
+		_, err := base64.StdEncoding.DecodeString(string(plainText))
+		if err != nil {
+			return keyActionRes, err
+		}
+		keysActionReq.PlainText = string(plainText)
+	}
+
+	if additionalAuthData != nil {
+		keysActionReq.AAD = *additionalAuthData
+	}
+
+	keysAction, err := c.doKeysAction(ctx, idOrAlias, "wrap", keysActionReq)
+	if err != nil {
+		return keyActionRes, err
+	}
+
+	keyActionRes = &KeyActionResponse{
+		PlainText:  keysAction.PlainText,
+		CipherText: keysAction.CipherText,
+		KeyVersion: KeyVersion{
+			ID:           keysAction.KeyVersion.ID,
+			CreationDate: keysAction.KeyVersion.CreationDate,
```

Comment:
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/50526396/206976513-e1651d2e-8977-41f4-821d-3000b15eb338.png">

RestAPI returns only the keyVersionID. Pls, confirm if we need to return the CreationDate as well.

Reply:
<img width="1686" alt="Screen Shot 2022-12-12 at 1 30 30 PM" src="https://user-images.githubusercontent.com/25618813/206992156-738bba5c-6961-4cdf-b06a-fcf8484b80e7.png">

The ActionResponse was created in reference with the apidocs, which mentions the creation Date, maybe there is a gap. Will revert for now.

Reply:
@dinesh-venkatraman-g ^^